pkg/http/responses.go

```diff
@@ -34,6 +34,19 @@ type BulkGetResponse struct {
 	Error    string            `json:"error,omitempty"`
 }
 
+// BulkPublishResponseEntry is an object representing a single entry in bulk publish response
+type BulkPublishResponseEntry struct {
+	EntryID string `json:"entryID"`
+	Status  string `json:"status"`
+	Error   string `json:"error,omitempty"`
+}
+
+// BulkPublishResponse is the response for bulk publishing events
+type BulkPublishResponse struct {
+	Statuses  []BulkPublishResponseEntry `json:"statuses"`
+	ErrorCode string                     `json:"errorCode,omitempty"`
+}
+
```

Comment:
Is it possible to reuse these from the contrib?

Reply:
No. Types are different in fields